### PR TITLE
Suggestions feedback

### PR DESCRIPTION
**Pull request by karanh37**
### Describe your changes:

image

I worked on ... because ...

#
### Type of change:

- [ ] Bug fix
- [ ] Improvement
- [ ] New feature
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/developers/contribute) document.
- [ ] My PR title is `Fixes <issue-number>: <short explanation>`
- [ ] I have commented on my code, particularly in hard-to-understand areas. 
- [ ] For JSON Schema changes: I updated the migration scripts or explained why it is not needed.

### CODE REVIEW

File: openmetadata-ui/src/main/resources/ui/src/components/Suggestions/SuggestionsSlider/SuggestionsSlider.tsx [hunk 1]
```diff
@@ -33,39 +35,47 @@ const SuggestionsSlider = () => {
...
               {t('label.reject-all')}
             </Typography.Text>
           </Button>
-        </>
+          <Button
+            ghost
+            className="exit-suggestion"
+            data-testid="close-suggestion"
+            icon={<CloseOutlined />}
+            onClick={() => onUpdateActiveUser()}
```
Comment (ShaileshParmar11) [inline openmetadata-ui/src/main/resources/ui/src/components/Suggestions/SuggestionsSlider/SuggestionsSlider.tsx:76]:
```suggestion
            onClick={onUpdateActiveUser}
```

File: openmetadata-ui/src/main/resources/ui/src/components/common/AvatarCarousel/AvatarCarousel.tsx [hunk 2]
```diff
@@ -43,48 +50,69 @@ const AvatarCarousel = () => {
...
-            key={avatar.id}
-            userName={avatar?.name ?? ''}>
+        {avatarList.map((avatar, index) => {
+          const isActive = currentSlide === index;
+          const button = (
             <Button
-              className={`p-0 m-r-xss avatar-item ${
-                currentSlide === index ? 'active' : ''
-              }`}
+              className={`p-0 m-r-xss avatar-item ${isActive ? 'active' : ''}`}
```
Comment (ShaileshParmar11) [inline openmetadata-ui/src/main/resources/ui/src/components/common/AvatarCarousel/AvatarCarousel.tsx:81]: please use classNames library for conditional className

File: openmetadata-ui/src/main/resources/ui/src/components/common/AvatarCarousel/AvatarCarousel.tsx [hunk 3]
```diff
@@ -43,48 +50,69 @@ const AvatarCarousel = () => {
...
-              <ProfilePicture name={avatar.name ?? ''} width="30" />
+              <ProfilePicture name={avatar.name ?? ''} width="28" />
             </Button>
-          </UserPopOverCard>
-        ))}
+          );
+
+          return (
+            <UserPopOverCard
+              className=""
```
Comment (ShaileshParmar11) [inline openmetadata-ui/src/main/resources/ui/src/components/common/AvatarCarousel/AvatarCarousel.tsx:90]: please remove className if not needed

File: openmetadata-ui/src/main/resources/ui/src/components/common/AvatarCarousel/avatar-carousel.less [hunk 4]
```diff
@@ -13,10 +13,10 @@
...
 
 .avatar-item {
-  opacity: 0.4;
-  &.active {
-    opacity: 1;
-  }
+  position: relative;
+  width: 28px !important;
+  height: 28px !important;
+  min-width: 28px !important;
```
Comment (ShaileshParmar11) [inline openmetadata-ui/src/main/resources/ui/src/components/common/AvatarCarousel/avatar-carousel.less:19]: why do we need important? can't we avoid it?

File: openmetadata-ui/src/main/resources/ui/src/components/common/AvatarCarousel/avatar-carousel.less [hunk 5]
```diff
@@ -29,4 +29,24 @@
...
+  .ant-btn {
+    padding: 0 12px;
+    height: 30px;
+  }
+  .exit-suggestion {
+    color: @grey-4 !important;
+    border-color: @grey-4 !important;
+    padding: 0 !important;
+    width: 30px;
+  }
```
Comment (ShaileshParmar11) [inline openmetadata-ui/src/main/resources/ui/src/components/common/AvatarCarousel/avatar-carousel.less:51]: same here as well

**Review by ShaileshParmar11 — COMMENTED** [review]
few comments

File: openmetadata-ui/src/main/resources/ui/src/components/Suggestions/SuggestionsSlider/SuggestionsSlider.tsx (hunk 1 above)
Comment (karanh37) [reply openmetadata-ui/src/main/resources/ui/src/components/Suggestions/SuggestionsSlider/SuggestionsSlider.tsx:76]: onUpdateActiveUser requires a parameter of type undefined or EntityRef, so need like this

**Review by ShaileshParmar11 — APPROVED** [review]

**Review by chirag-madlani — DISMISSED** [review]

**Review by chirag-madlani — APPROVED** [review]

**Pull request merged**